profiles/common/modules/features/communities/communities.module

```diff
@@ -335,55 +355,55 @@ function _communities_get_url_in_group($path, $original_path) {
 }
 
 
-
-/*
-* _to_arg function
-*/
-
-
-function group_name_to_arg($arg=NULL) {
+/**
+ * Implementation of _to_arg function.
+ */
+function group_name_to_arg($arg = NULL) {
   module_load_include('inc', 'pathauto', 'pathauto');
   $context = og_context();
   if (isset($context['gid'])) {
     $node = node_load($context['gid']);
     return pathauto_cleanstring($node->title);
   }
-  else
+  else {
     return 'none';
+  }
 }
 
 
-/*
-* _load function
-*/
+/**
+ * Implementation of _load function.
+ */
 function group_name_load($arg) {
   global $language;
```

Comment:
A `_load()` function should be fully documented. See for example `node_load()` and `user_load()`. This function has a very unfortunate name, since it does not actually load a "group name", instead it loads the group _node_ but then only returns the node ID instead of the full node :-/

Anyway let's just focus on the coding standards.

For example:

```
/**
 * Returns the ID of the Community node corresponding to the given group name.
 *
 * @param string $arg
 *   The group name, or 'none' if no ID should be returned.
 *
 * @return string | 0
 *   The node ID of the Community node, or the integer 0 if no group name was given.
 */
```